velox/functions/prestosql/GreatestLeast.h

```diff
@@ -66,6 +66,11 @@ struct ExtremeValueFunction {
     return wrapper;
   }
 
+  int128_t extractValue(
```

Comment:
I may have tried this and it didn't work because it was ambiguous, or it may not have occurred to me it would be needed, but could we consolidate these as:

  template <typename U>
  U extractValue(
      const exec::CustomTypeWithCustomComparisonView<U>& wrapper) const {
     return *wrapper;
   }

Reply:
Also, can you please add a test case for this?

Reply:
Good catch